docs/designers-developers/developers/data/data-core.md

```diff
@@ -153,21 +153,6 @@ _Returns_
 
 -   `?Object`: Record.
 
-<a name="getEntityRecordChangesByRecord" href="#getEntityRecordChangesByRecord">#</a> **getEntityRecordChangesByRecord**
```

Comment:
This selector was very hard to work with and I checked wpdirectory.net it's not something that is used by plugins and is unlikely to be used before FSE.